### ISSUE TITLE
Upgrade outdated dependencies

### ISSUE THREAD

**Issue by bintzandt** [Yoast/javascript#626]
## Explanation

In the monorepo, there are some dependencies that are outdated. We should upgrade them in the future to ensure that bugs are fixed and we can profit from new functionality. 

There are also some packages that will need to be upgraded in order to have node v12 support (which is the current LTS release).

The list of outdated packages can be found using `yarn outdated`.

## Technical decisions

* `babel-cli`, `webpack-dev-server` and `jest` are the ones that *need* to be upgraded before we can have proper node v12 support.
* Remove the `resolutions` from `package.json` when the upgrade is complete since it is no longer needed.
